cmd/kube-apiserver/app/options/options.go

```diff
@@ -33,6 +33,8 @@ type APIServer struct {
 	*genericapiserver.ServerRunOptions
 	AllowPrivileged             bool
 	EventTTL                    time.Duration
+	KeystoneConfig              string
+	KeystoneAuthMode            string
```

Comment:
By the way, I still think keep the original `KeystoneUrl` and add another field `KeystoneTokenAuthnConfigFile` would be better. Forget about the password authenticator when you are developing a new.